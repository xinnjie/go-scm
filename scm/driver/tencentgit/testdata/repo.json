{
    "id": 179129,
    "description": "a repo for tencentgit client integration test",
    "public": false,
    "archived": false,
    "visibility_level": 10,
    "public_visibility": 100,
    "namespace": {
        "created_at": "2021-08-12T12:16:46+0000",
        "description": "wx_1600e990b7cb469a97087b334c94acee",
        "id": 228317,
        "name": "xinnjie",
        "owner_id": 176405,
        "path": "xinnjie",
        "updated_at": "2021-08-12T12:16:46+0000"
    },
    "owner": {
        "id": 176405,
        "username": "xinnjie",
        "web_url": "https://git.code.tencent.com/u/xinnjie",
        "name": "xinnjie",
        "state": "active",
        "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "name": "testme",
    "name_with_namespace": "xinnjie/testme",
    "path": "testme",
    "path_with_namespace": "xinnjie/testme",
    "default_branch": "master",
    "ssh_url_to_repo": "git@git.code.tencent.com:xinnjie/testme.git",
    "http_url_to_repo": "http://git.code.tencent.com/xinnjie/testme.git",
    "https_url_to_repo": "https://git.code.tencent.com/xinnjie/testme.git",
    "web_url": "https://git.code.tencent.com/xinnjie/testme",
    "tag_list": [],
    "issues_enabled": true,
    "merge_requests_enabled": true,
    "wiki_enabled": true,
    "snippets_enabled": true,
    "review_enabled": true,
    "fork_enabled": true,
    "tag_name_regex": null,
    "tag_create_push_level": 30,
    "branch_name_regex": null,
    "created_at": "2021-08-20T10:44:47+0000",
    "last_activity_at": "2021-08-22T16:29:28+0000",
    "creator_id": 176405,
    "avatar_url": "https://git.code.tencent.com/uploads/project/avatar/179129",
    "watchs_count": 1,
    "stars_count": 0,
    "forks_count": 0,
    "config_storage": {
        "limit_lfs_file_size": 5120.0,
        "limit_size": 5120.0,
        "limit_file_size": 128.0,
        "limit_lfs_size": 512000.0
    },
    "forked_from_project": "Forked Project not found",
    "statistics": {
        "commit_count": 3,
        "repository_size": 0.004
    },
    "suggestion_reviewers": [],
    "necessary_reviewers": [],
    "path_reviewer_rules": null,
    "approver_rule": 1,
    "necessary_approver_rule": 0,
    "can_approve_by_creator": false,
    "auto_create_review_after_push": false,
    "forbidden_modify_rule": false,
    "push_reset_enabled": true,
    "merge_request_template": null,
    "file_owner_path_rules": null
}