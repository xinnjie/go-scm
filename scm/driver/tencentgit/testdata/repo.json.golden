{
    "ID": "179129",
    "Namespace": "xinnjie",
    "Name": "testme",
    "FullName": "xinnjie/testme",
    "Perm": {
        "Pull": true,
        "Push": false,
        "Admin": false
    },
    "Branch": "master",
    "Private": false,
    "Clone": "https://git.code.tencent.com/xinnjie/testme.git",
    "CloneSSH": "git@git.code.tencent.com:xinnjie/testme.git",
    "Link": "https://git.code.tencent.com/xinnjie/testme",
    "Created": "2021-08-20T10:44:47Z",
    "Updated": "0001-01-01T00:00:00Z"
}
